antiplag/tests.py

```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
+from django.core.mail import send_mail
+from django.conf import settings
+# Create your tests here.
+
+class EmailTestCase(TestCase):
+
+    def test_sending(self):
+        """Verify that send_mail function works"""
+        self.assertEqual(send_mail('testing', 'my message', 'hello@mydomain.com', ['personal@example.com'], fail_silently=False), 1)
```

Comment:
Well this is kinda dumb test case - it just tests third party package whether it works? I think these dependencies have their own testing. It is unable to check the delivery, therefore it has no meaning.

I would suggest creating a test for endpoint - what happens if we submit request without email, email as empty or incorrect string, and email in correct format.